cuda_bindings/tests/test_cuda.py

```diff
@@ -980,12 +980,14 @@ def test_all_CUresult_codes():
     assert num_good >= 76  # CTK 11.0.3_450.51.06
 
 
+@pytest.mark.skipif(driverVersionLessThan(12030), reason="Driver too old for cuKernelGetName")
```

Comment:
Instead of skipping should we use the driver version check to ensure that we're throwing an exception as expected?

Reply:
Tracked in #728